docs/DeveloperGuide.md

```diff
@@ -740,58 +739,109 @@ testers are expected to do more *exploratory* testing.
-
-1. Saving window preferences
-
+2. Saving window preferences
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
-
-   1. Re-launch the app by double-clicking the jar file.<br>
+   2. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
-
 ### Adding a person
 
+1. Adding a person
+   1. Prerequisites: None
+   2. Test case:`add person n/Elmo p/98765432 e/elmo@sesa.me a/Sesame Street st/recovering t/insurance`
+   3. Expected: A new contact is added to the list.
+   - The status bar shows the following: "New person added: Elmo; Phone: 98765432; 
+   Email: elmo@sesa.me; Address: Sesame Street; Status: recovering; Tags: [insurance]"
+   - The list of persons should now have the new contact inside it.
+    
+
 ### Deleting a person
 
 1. Deleting a person while all persons are being shown
-
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
-
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
-
-   1. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
-
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
-
-1. _{ more test cases …​ }_
+   1. Prerequisites: Multiple persons in the DocTrack. If they are not shown, do `list person`.
+   2. Test case: `delete person 1`<br>
+      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. 
+   Timestamp in the status bar is updated.
+   3. Test case: `delete person -1`<br>
+         Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+   4. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+         Expected: Similar to previous.
 
 ### Editing a person
 
+1. Editing a person
+    1. Prerequisites: At least one person is showing in the list.
+    2. Test case:`edit person 1 p/90909090`
+    3. Expected: The contact is edited.
+    - The status bar shows the following: "Edited person.... Phone: 90909090...."
+    - The list of persons should now have the edited contact inside it.
+
 ### Finding a person
 
+1. Finding a person
+    1. Prerequisites: At least one person in DocTrack. 
+        - For a simple example, use the person added in the test case for adding a person.
+    2. Test case:`find person n/Elmo`
+    3. Expected: The list is updated to show all the people with names containing "elmo", non-case-sensitive.
+    - The status bar shows the following: "Found _ persons".
```

Comment:
Would it be better to include a bracket to explain where `_ is the number of persons with names containing "elmo"`?

Reply:
Good idea.